lumicks/pylake/kymo.py

```diff
@@ -483,6 +483,16 @@ def pixelcount_factory(_):
         result._pixelcount_factory = pixelcount_factory
         return result
 
+    def flip(self):
```

Comment:
What happens if you flip multiple times?
What happens if I use multiple but different image factories in sequence, say,

1. downsample,
2. flip,
3. downsample.

Reply:
Double flip is explicitly tested. Downsample should also be fine (I can easily add a specific test for it, but it seems that it should be fine considering the double flip tests the factories in sequence already).